sympy/calculus/util.py

```diff
@@ -37,6 +47,16 @@ def continuous_domain(f, symbol, domain):
     >>> continuous_domain(log(2*x - 1), x, S.Reals)
     Interval.open(1/2, oo)
 
+    Returns
+    =======
+    Intervals where the function is continuous.
+
+    Raises
+    ======
+    NotImplementedError
+        if the method to determine continuity of such a function
+        has not yet been developed
```

Comment:
I think it is best to make these sentences starting with capital letters and ending with periods. Make sure all are consistent.